### ISSUE TITLE
audio-recorder-streamlit 0.0.9 isn't compatible with newer streamlit version

### ISSUE THREAD

**Issue by terrywu1000** [Joooohan/audio-recorder-streamlit#19]
For Ubuntu 22.04,   I found audio-recorder-streamlit 0.0.9 isn't compatible with newer streamlit version (e.g. streamlit 1.34.0). So with streamlit 1.34.0, the mic icon of audio-recorder-streamlit  doesnt' show up at all. But I have to use new features from new streamlit. 

Any plan to release new version of audio-recorder-streamlit?

> Installing collected packages: streamlit
>   Attempting uninstall: streamlit
>     Found existing installation: streamlit 1.29.0
>     Uninstalling streamlit-1.29.0:
>       Successfully uninstalled streamlit-1.29.0
> ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
> audio-recorder-streamlit 0.0.9 requires streamlit<1.30,>=0.63, but you have streamlit 1.34.0 which is incompatible.
> Successfully installed streamlit-1.34.0

**Comment by touhi99** [comment]
+ Same issue

**Comment by djk1983** [comment]
I would welcome being able to use this with Streamlit 1.35.0

**Comment by lkdd-ao** [comment]
I won't solve this problem by fixing the original code, but you can try using the following method 👇
```
import streamlit as st

st.html(
    '''
     <style>
        iframe[title="audio_recorder_streamlit.audio_recorder"] {
            height: auto;
        }
     </style>
    '''
)
```

**Comment by Joooohan** [comment]
Hi, I have tested with 1.37.0 and this seems to work. I will release a new version releasing the constraint.

**Comment by Joooohan** [comment]
Should be fixed in 0.0.10.
